programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md

```diff
@@ -264,6 +264,24 @@ fundamentals, familiarity with [sigstore](https://www.sigstore.dev/),
     - Frances Thai @notfelineit frances@planetscale.com
 - Upstream Issue (URL): https://github.com/vitessio/arewefastyet/issues/525
 
+### TUF
+
+#### Documentation assessment and improvements
+
+- Description:
+  - Open source projects need help with their documentation!  The TUF project is a good place to start.  We'd welcome help from others to help here
+and become contributors to other projects / TAGs later in the project period.  The mentee will (with minimal guidance from the CNCF team and TUF project) do a CNCF assessment for the TUF documentation
```

Comment:
If there's a tutorial/guide available for doing the assessment, please link here. It would help mentee candidates a lot to understand what they're getting into.